wallet/how-to/use-3rd-party-integrations/js-infura-api.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 2
+description: Use Infura and custom nodes to make direct, read-only requests.
+---
+
+# Make read-only requests from your JavaScript dapp
+
+You can use the [Infura API](https://docs.infura.io/) from your JavaScript dapp with
+[MetaMask SDK](../connect/set-up-sdk/javascript/index.md) installed to make direct, read-only
+JSON-RPC requests.
```

Comment:
We can probably delete this paragraph. It's covered by the prerequisites, and the paragraph preceding the prerequisites.

Reply:
I wanted to lead with this info since it highlights the third-party integration (Infura API) and the applicable platform (JS dapps using the SDK)